src/control/lib/daos/pool_cont_prop.go

```diff
@@ -235,25 +234,11 @@ func RpPdaIsValid(pda uint64) bool {
 	return bool(C.daos_rp_pda_valid(C.uint32_t(pda)))
 }
 
-// PoolPolicy defines a type to be used to represent DAOS pool policies.
-type PoolPolicy uint32
-
-const (
-	// PoolPolicyIoSize sets the pool's policy to io_size
-	PoolPolicyIoSize PoolPolicy = C.DAOS_MEDIA_POLICY_IO_SIZE
-	// PoolPolicyWriteIntensivity sets the pool's policy to write_intensivity
-	PoolPolicyWriteIntensivity PoolPolicy = C.DAOS_MEDIA_POLICY_WRITE_INTENSIVITY
-)
-
-// PoolPolicyIsValid returns a boolean indicating whether or not the
-// pool tiering policy string is valid.
-func PoolPolicyIsValid(polStr string) bool {
-	var polDesc C.struct_policy_desc_t
-
-	cStr := C.CString(polStr)
-	defer C.free(unsafe.Pointer(cStr))
-
-	return bool(C.daos_policy_try_parse(cStr, &polDesc))
+func DataThreshIsValid(size uint64) bool {
```

Comment:
nit: exported (start with uppercase letter) functions require a preceding comment e.g. `// DataThreshIsValid verifies that the input value meets the required criteria..`